src/reference/02-DetailTopics/03-Dependency-Management/03-Library-Management.md

```diff
@@ -296,6 +296,21 @@ libraryDependencies +=
 
 See [ModuleID](../api/sbt/ModuleID.html) for API details.
 
+In certain cases a transitive dependency should be exluded from
+all dependencies. This can be achieved by setting up `SbtExclusionRules`
+in `excludeDependencies`. 
+
+```scala
+excludeDependencies ++= Seq(
+  // commons-logging is replaced by jcl-over-slf4j
+  SbtExclusionRule("commons-logging", "commons-logging")
+)
+```
+
+Available since sbt 0.13.8.
```

Comment:
It's been available since 0.13.8 as an experimental feature, but Maven compatibility was added in 0.13.9 - https://github.com/sbt/sbt/pull/2035